materials/section-4/README.md

```diff
@@ -0,0 +1,87 @@
+# Database (Extra Credit)
+
+In this extra credit section we will add a database and integrate it with our app. Storing state in a atom is perfectly fine if you don't want to have persistent data, otherwise you probably need a datastore or database. We will be using _postgres_ as our database. We won't be talking about specific databases in this section, only how you integrate with them (in this case postgres).
```

Comment:
Some nitpicking from a non-native speaker. I could be completely wrong here :) 

```suggestion
In this extra credit section we will add a database and integrate it with our app. Storing state in a atom is perfectly fine if you don't want to have persistent data, otherwise you would probably need a datastore or database. We will be using _postgres_ as our database. We won't be talking about specific databases in this section, only how you integrate with them (in this case postgres).
```